config/clients/python/template/README_calling_api.mustache

```diff
@@ -245,6 +245,7 @@ Reads the list of historical relationship tuple writes and deletes.
 options = {
     # You can rely on the model id set in the configuration or override it for this specific request
     "page_size": "25",
+    "start_time": "2022-01-01T00:00:00Z",
```

Comment:
I think this needs to move as part of the `ClientReadChangesRequest` on line 251?